lib/reek/report/report.rb

```diff
@@ -144,53 +144,46 @@ class XMLReport < Base
       require 'rexml/document'
 
       def show
-        checkstyle = REXML::Element.new('checkstyle', document)
-
-        smells.group_by(&:source).each do |file, file_smells|
-          file_to_xml(file, file_smells, checkstyle)
-        end
-
-        print_xml(checkstyle.parent)
+        document.write $stdout, 2
```

Comment:
Magic number alert..;)
I know what the 2 refers to, but it is still baroque.
Any idea how to make this clearer?